keps/sig-instrumentation/1933-secret-logging-static-analysis/kep.yaml

```diff
@@ -0,0 +1,18 @@
+title: Defend against accidental credential logging via static analysis interation into Prow.
```

Comment:
:wave: Enhancement shadow here!

May i ask you to update this `kep.yaml` with the missing fields from the template: https://github.com/kubernetes/enhancements/blob/master/keps/NNNN-kep-template/kep.yaml.

And don't forget to fill in the `reviewers` and `approvers` fields that are `TBD` at the moment!